examples/python/gepetto-viewer.py

```diff
@@ -8,10 +8,15 @@
 import pinocchio as pin
 import numpy as np
 import os
-from future.builtins import input
 
 from pinocchio.visualize import GepettoVisualizer
 
+try:
```

Comment:
This is a bit manual to do it like this in order to have `input` mean the same thing on Python 2 and Python 3 but you don't have the additional dependency on the `future` package.

Reply:
Sounds good to me. @lesteve Thanks for the fixes.